docs/docs/features/hardware-transcoding.md

```diff
@@ -100,6 +100,20 @@ immich-microservices:
 
 Once this is done, you can continue to step 3 of "Basic Setup".
 
+#### Immich Docker for Intel QuickSync - Unraid Setup 
```

Comment:
```suggestion
#### Intel QuickSync
```

Reply:
It should just be `QSV` to be consistent with the other headers.